trino/trino.go

```diff
@@ -725,7 +725,7 @@ type stmtErrorFailureInfo struct {
 }
 
 func (e stmtError) Error() string {
-	return e.FailureInfo.Type + ": " + e.Message
+	return fmt.Sprintf("%v (name: %s, code: %d): %s", e.FailureInfo.Type, e.ErrorName, e.ErrorCode, e.Message)
```

Comment:
Can you add a simple test for that? Maybe a query with a syntax error? I'd like to see some example value and make sure it's human-readable and informative.

We should not encourage parsing this string and instead fix the interfaces as suggested in #97.

Reply:
Can do, but wouldn't a test encourage folk to rely on the message as if it was some sort of contract 🤔 ?

Reply:
PTAL at #109, maybe it solves getting error details without having to parse the error string.